beacon-chain/blockchain/block_processing.go

```diff
@@ -323,13 +323,22 @@ func (c *ChainService) runStateTransition(
 // validators were activated from current epoch. After it saves, current epoch key
 // is deleted from ActivatedValidators mapping.
 func (c *ChainService) saveValidatorIdx(state *pb.BeaconState) error {
-	activatedValidators := validators.ActivatedValFromEpoch(helpers.CurrentEpoch(state) + 1)
+	nextEpoch := helpers.CurrentEpoch(state) + 1
```

Comment:
lets use the `NextEpoch` helper here

Reply:
There's no more `NextEpoch` in the 0.6 spec, it needs to be deleted hence I try to avoid using it. Not a big fan of `NextEpoch`, it doesn't add much value, it's just `CurrentEpoch + 1` and we barely use it